packages/block-library/src/post-excerpt/edit.js

```diff
@@ -99,21 +100,36 @@ export default function PostExcerptEditor( {
 	const excerptClassName = classnames( 'wp-block-post-excerpt__excerpt', {
 		'is-inline': ! showMoreOnNewLine,
 	} );
+
+	/*
+	 * translators: If your word count is based on single characters (e.g. East Asian characters),
+	 * enter 'characters_excluding_spaces' or 'characters_including_spaces'. Otherwise, enter 'words'.
+	 * Do not translate into your own language.
+	 */
+	const wordCountType = _x( 'words', 'Word count type. Do not translate!' );
```

Comment:
Perhaps the `Word count type. Do not translate!` text should include some extra info to convey the message that this should be one of `words`, `characters_excluding_spaces`, or `characters_including_spaces`? Right now nobody will know what to do, unless they search the source code to see the code comment 🤔